### DISCUSSION TITLE
Gruppenfunktion in den 3.71.8 Nightly's defekt

### DISCUSSION THREAD

**Discussion by Baxxy13** [OpenCCU/OpenCCU#2363] [🇩🇪 Fragen&Antworten] [answered]
Defekt ist eher die Anzeige in der WebUI, bestehende Gruppen funktionieren aber weiterhin.

- Alle bestehenden Gruppen werden mit "Es sind keine Geräte in der Gruppe" angezeigt.
- HM-Gruppen bieten nur IP-Geräte zum Hinzufügen an

Weiter habe ich nicht geguckt/getestet da ich mir meine Gruppen nicht zerlegen wollte.

**Comment by jens-maus** [comment]
Das kann ich nach einem kurzen Test so bestätigen. Auch bei mir werden bei "Bearbeiten" in der Auflistung der Gruppengeräte einfach nur "Es sind keine Geräte in der Gruppe" dargestellt. Auch sehe ich das bereits existierende BidCos/nicht-HmIP Heizungsgruppen trotzdem oben bei der Auswahl dann "HmIP-Heizungsgruppe" anzeigen obwohl das keine HmIP-Heizungsgruppe ist und dort folglich als Auflistung auch keine BidCos-Geräte angeboten werden ("Badezimmer" ist hier z.B. eine BidCos und keine HmIP-Heizungsgruppe):

![Bildschirmfoto 2023-07-17 um 09 03 45](https://github.com/jens-maus/RaspberryMatic/assets/5887769/f6485071-0578-4efc-8a89-3aab163aea6a)

Scheint also wirklich in der 3.71.8 ein entsprechendes Problem zu geben. Die Frage wäre nur RaspberryMatic-only oder auch bei der CCU3 dann sichtbar?!?!

@Baxxy13 vllt. kannst du dazu mal ein entsprechendes Issue-Ticket dazu aufmachen und das vllt. auch mal auf einer vanilla Testinstallation probieren, etc.? Danke.

**Reply by jp112sdl** [reply]
Hab kein Testsystem, aber ich denke mal,  diese 2 unspektakulär aussehenden Änderungen in der `GroupEditPage.ftl` haben es kaputt gemacht...
https://github.com/eq-3/occu/commit/600af4b39109022f0cd9db18d4acf8f299120a56#diff-15c5c06942852c0dbe1941d80127a96027694a1f8def42671ebedc5226349351

**Reply by jens-maus** [reply]
Danke für die Rückmeldung! Das wird es vmtl. sein. eQ3 hat es auch bereits bestätigt und der nächste kommende RC von OCCU wird es vmtl. wieder beheben... So zumindest die aktuellen Infos.

**Comment by jens-maus** [comment, marked as answer]
Bitte mal mit der aktuellsten 3.71.11.20230719-6d6423 Testversion probieren. Nun sollte das Problem behoben sein.

**Reply by Baxxy13** [reply]
Ja, das geht wieder.
Dafür können keine HmIP-Direktverknüpfungen mehr "bearbeitet" werden.
Keine Ahnung wo das nun wieder herkommt.
